packages/hadron-document/src/element.ts

```diff
@@ -70,11 +59,9 @@ export class Element extends EventEmitter {
   currentValue: BSONValue;
   added: boolean;
   removed: boolean;
-  elements?: LinkedList | null;
+  elements?: ElementList;
   originalExpandableValue?: BSONObject | BSONArray;
   parent: Element | Document | null;
-  previousElement: Element | null;
-  nextElement: Element | null;
```

Comment:
[Suggestion, feel free to ignore:] Would it make sense to add getters for these (based on `parent.elements.findNext()`, like the replacements in the test file above)? I don’t see them being used a lot outside of tests but they seem convenient.

Reply:
Totes, will add

Reply:
Done in 0ccce28cda98ae631601081ff5ca7b6b562e9f3a